src/mcp/tools/auth/set_auth_claims.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { setCustomClaim } from "../../../gcp/auth.js";
+
+export const set_auth_claims = tool(
+  {
+    name: "set_auth_claims",
+    description: "Sets a list of specific claims for a user.",
```

Comment:
I've seen advice that these should be more descriptive.

```suggestion
    description: "Sets custom claims on a specific user's account. Use to create trusted values associated with a user e.g. marking them as an admin. Claims are limited in size and should be succinct in name and value.",
```